src/VecSim/vec_sim_common.h

```diff
@@ -17,21 +17,37 @@ extern "C" {
 #include <stdbool.h>
 
 // Common definitions
-#define INVALID_ID    UINT_MAX
-#define INVALID_LABEL SIZE_MAX
-#define UNUSED(x)     (void)(x)
+#define DEFAULT_BLOCK_SIZE 1024
+#define INVALID_ID         UINT_MAX
+#define INVALID_LABEL      SIZE_MAX
+#define UNUSED(x)          (void)(x)
 
 // HNSW default parameters
 #define HNSW_DEFAULT_M       16
 #define HNSW_DEFAULT_EF_C    200
 #define HNSW_DEFAULT_EF_RT   10
 #define HNSW_DEFAULT_EPSILON 0.01
-#define DEFAULT_BLOCK_SIZE   1024
 
 #define HNSW_INVALID_LEVEL SIZE_MAX
 #define INVALID_JOB_ID     UINT_MAX
 #define INVALID_INFO       UINT_MAX
 
+// SVS-Vamana default parameters
+#define SVS_VAMANA_DEFAULT_ALPHA_L2                 1.2f
+#define SVS_VAMANA_DEFAULT_ALPHA_IP                 0.95f
+#define SVS_VAMANA_DEFAULT_GRAPH_MAX_DEGREE         32
+#define SVS_VAMANA_DEFAULT_CONSTRUCTION_WINDOW_SIZE 200
+#define SVS_VAMANA_DEFAULT_MAX_CANDIDATE_POOL_SIZE  (3 * SVS_VAMANA_DEFAULT_CONSTRUCTION_WINDOW_SIZE)
```

Comment:
what if SVS_VAMANA_DEFAULT_CONSTRUCTION_WINDOW_SIZE is defined by the user differently than the default, but SVS_VAMANA_DEFAULT_MAX_CANDIDATE_POOL_SIZE is not?

Reply:
Do you mean it shouldn’t be configurable at all, or just not exposed via the info command?
In any case, if we do expose it in the info output, I think it should reflect the correct runtime-derived value. So maybe the options are:
1. Don’t expose it now and possibly introduce a runtime fetch API later if needed, or
2. Remove it altogether if it's not meant to be user-facing.